test/JDBC/expected/TestSpatialFun-vu-verify.out

```diff
@@ -0,0 +1,932 @@
+-- For STEquals
+DECLARE @point1 geometry = geometry::Point(1.0, 2.0, 4326);
+DECLARE @point2 geometry = geometry::Point(3.0, 4.0, 4326);
+SELECT @point1.STEquals(@point2) AS Equal;
+go
+~~START~~
+bit
+0
+~~END~~
+
+
+DECLARE @point1 geometry = geometry::Point(1.0, 2.0, 4326);
+DECLARE @point2 geometry = geometry::Point(3.0, 4.0, 4326);
+SELECT STEquals(@point1, @point2);
+go
+~~START~~
+bit
+0
+~~END~~
+
+
+DECLARE @point1 geometry = geometry::Point(1.0, 2.0, 4326);
+DECLARE @point2 geometry = geometry::Point(3.0, 4.0, 4326);
+SELECT @point1 . STEquals ( @point2 );
+go
+~~START~~
+bit
+0
+~~END~~
+
+
+SELECT PointColumn1.STEquals(PointColumn2) AS Equals FROM YourTableTemp2;
+go
+~~START~~
+bit
+0
+~~END~~
+
+
+DECLARE @point1 geometry = geometry::Point(1.0, 2.0, 4326);
+SELECT PointColumn1.STEquals(@point1) AS Equals FROM YourTableTemp2;
+go
+~~START~~
+bit
+0
+~~END~~
+
+
+DECLARE @point1 geometry = geometry::Point(1.0, 2.0, 4326);
+SELECT @point1.STEquals(PointColumn2) AS Equals FROM YourTableTemp2;
+go
+~~START~~
+bit
+0
+~~END~~
+
+
+DECLARE @referencePoint geometry = geometry::Point(0.0, 0.0, 4326);
+DECLARE @isEqual BIT = 1;
+SELECT * FROM YourTableTemp WHERE PointColumn.STEquals(@referencePoint) = @isEqual;
+go
+~~START~~
+int#!#geometry
+~~END~~
+
+
+SELECT ID, PointColumn1.STEquals(PointColumn2) AS Equal_points FROM YourTableTemp2;
+go
+~~START~~
+int#!#bit
+1#!#0
+~~END~~
+
+
+SELECT * FROM TableATemp JOIN TableBTemp ON PointA.STEquals(TableBTemp.PointB) <= 1;
```

Comment:
1. What are trying to test here? `STEquals` returns binary so it can only return 1 or 0 so it means that the join condition will return all the rows. The join condition can simply be `ON PointA.STEquals(TableBTemp.PointB)` or `ON PointA.STEquals(TableBTemp.PointB) != 1`.
2. It's better to print the geograph/geometry columns in plain text using `STAsText` instead of raw binary data.

Reply:
Resolved